docs/how-to/configure-jenkins-memory-usage.md

```diff
@@ -0,0 +1,13 @@
+# How to control heap memory of the jenkins-k8s-operator charm
```

Comment:
I think it might help to match the title with the file name :)

Reply:
Good point, I'll change this in a follow-up PR